<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HIGHTLIGHT KEYWORDS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_select_file">
         <property name="text">
          <string>选择 文件 (F)</string>
         </property>
         <property name="shortcut">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_select_folder">
         <property name="text">
          <string>选择 文件夹 (Shift + F)</string>
         </property>
         <property name="shortcut">
          <string>Shift+F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_selected_path">
       <property name="text">
        <string>已选择：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_selected_path">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_strategy">
       <property name="text">
        <string>高亮策略：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_strategy">
       <item>
        <property name="text">
         <string>1. Markdown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2. Text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3. HTML</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_edit_keywords">
       <property name="text">
        <string>编辑 高亮关键字 (E)</string>
       </property>
       <property name="shortcut">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_execute">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>执行 (Enter)</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_info">
       <property name="text">
        <string>信息：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
